content/exchange/artifacts/Generic.Events.Processes.yaml

```diff
@@ -0,0 +1,22 @@
+name: Generic.Events.Processes
```

Comment:
This is basically what the process tracker does a lot better :-)

Reply:
Is the process tracker available for all supported operating systems? Ive only seen it for Windows.

Reply:
yes it is available always - you can read more about it here 

https://docs.velociraptor.app/blog/2022/2022-08-17-process-tracker/

https://docs.velociraptor.app/artifact_references/pages/linux.events.trackprocesses/

This artifact is most similar to your idea since it only uses pslist
https://docs.velociraptor.app/artifact_references/pages/windows.events.trackprocessesbasic/

This one uses sysmon events as well
https://docs.velociraptor.app/artifact_references/pages/windows.events.trackprocesses/